keras_nlp/layers/modeling/masked_lm_head_test.py

```diff
@@ -12,80 +12,53 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-
-from keras_nlp.backend import keras
 from keras_nlp.backend import ops
 from keras_nlp.layers.modeling.masked_lm_head import MaskedLMHead
 from keras_nlp.layers.modeling.reversible_embedding import ReversibleEmbedding
 from keras_nlp.tests.test_case import TestCase
 
 
 class MaskedLMHeadTest(TestCase):
-    def test_valid_call(self):
-        head = MaskedLMHead(
-            vocabulary_size=100,
-            activation="softmax",
+    def test_basics(self):
```

Comment:
Should we call these tests something better than `test_basics` perhaps?

I don't love that the tests bury so many assertions in them but maybe we can't avoid it. At the very least it could be nice to call them something like `test_layer_behaviors`

Reply:
No strong feeling on the name. My main thinking was just to follow keras-core here.

https://github.com/keras-team/keras-core/blob/ac9be3319e8a907e4db4348421a6488e258517a2/keras_core/layers/core/embedding_test.py#L9-L34

Even if it's not perfect, the continuity seemed like a good thing. We could try to have some test suite with test names, but why get so fancy? I haven't minded the keras-core setup practically.

Reply:
Ack, fine with me. As long as it's not practically hurting test debug-ability in your experience then I am okay with it